demos/http/http_demo_s3_download/http_demo_s3_download.c

```diff
@@ -49,14 +58,14 @@
     #error "Please define a HTTPS_PORT."
 #endif
 
-/* Check that a path for Root CA Certificate is defined. */
+/* Check that a path for Root CA Certificate is defined for AWS IOT SERVICE. */
 #ifndef ROOT_CA_CERT_PATH
-    #error "Please define a ROOT_CA_CERT_PATH."
+    #error "Please define a ROOT_CA_CERT_PATH for AWS IOT."
 #endif
 
-/* Check that the pre-signed GET URL is defined. */
-#ifndef S3_PRESIGNED_GET_URL
-    #error "Please define a S3_PRESIGNED_GET_URL."
+/* Check that a path for Root CA Certificate is defined for AWS S3 Service. */
+#ifndef ROOT_CA_CERT_PATH_S3
```

Comment:
To be consistent with other naming macros that use the `AWS_S3` prefix
```suggestion
#ifndef AWS_S3_ROOT_CA_CERT_PATH
```